python/cudf/cudf/core/column/column.py

```diff
@@ -1750,6 +1815,8 @@ def as_column(
     nan_as_null: bool = None,
     dtype: Dtype = None,
     length: int = None,
+    copy: bool = False,
+    fastpath: bool = False,
```

Comment:
Can we avoid adding these arguments to `as_column`? See my comment on `Series.copy()` below.

Reply:
For this PR: Yes. I have removed these for now. Will explain why and the future potential for these in `Series.copy` comment below.